### ISSUE TITLE
Alt-Text for Images

### ISSUE THREAD

**Issue by Viomi** [misskey-dev/misskey#6687]
## Summary

Currently, there is no way to add alt-text for images. This is a serious flaw for accessibility, as it means blind users literally cannot understand any of the images posted on any misskey instance.

My suggestion is we add a button to allow you to add "alt-text" to an image, so people using screen-readers can still understand the content.

**Comment by Viomi** [comment]
https://github.com/syuilo/misskey/pull/6601

Looks like it's already being worked on :)
